### PR TITLE
Add HTML Tidy License

### PR DESCRIPTION
**Pull request by johnmhoran**
Add an `.xml` and `.txt` file for the new `HTML Tidy License`.

### CODE REVIEW

**Comment by johnmhoran** [comment]
@swinslow @jlovejoy Neglected to add you as reviewers when I opened this PR -- looks like the 2 Travis CI jobs finally started and succeeded.  Please let me know if you have any comments, edit requests etc.

File: src/HTMLTIDY.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="HTMLTIDY" name="HTML Tidy License">
```
Comment (swinslow) [inline src/HTMLTIDY.xml:3]: @johnmhoran, can you update this `license` tag to include `listVersionAdded="3.11"` ?

**Comment by swinslow** [comment]
Thanks @johnmhoran! This looks great, I think the only update needed is to add the `listVersionAdded` property that I mentioned in the comment above. After that it should be all set to merge.

File: src/HTMLTIDY.xml (hunk 1 above)
Comment (johnmhoran) [reply src/HTMLTIDY.xml:3]: Will do @swinslow -- thank you.  👍

**Comment by swinslow** [comment]
Thanks @johnmhoran! I'll merge after the tests pass.

**Pull request merged**